src/Epsagon.Dotnet.Lambda/EpsagonHandler.cs

```diff
@@ -28,20 +31,20 @@ public class EpsagonHandler
             {
 
                 Utils.DebugLogIfEnabled("entered epsagon lambda handler");
-
+                
                 // handle trigger event
                 using (var scope = GlobalTracer.Instance.BuildSpan("").StartActive(finishSpanOnDispose: true))
                 {
                     var trigger = TriggerFactory.CreateInstance(input.GetType(), input);
                     trigger.Handle(context, scope);
                 }
-
+       
                 // handle invocation event
                 using (var scope = GlobalTracer.Instance.BuildSpan((typeof(TEvent).Name)).StartActive(finishSpanOnDispose: true))
                 using (var handler = new LambdaTriggerHandler<TEvent, TRes>(input, context, scope))
                 {
                     handler.HandleBefore();
-
+                    setTraceUrl(handler.getTraceUrl());
```

Comment:
access local methods using `this` for consistency

Reply:
I can't do it from a static method, fixed on LambdaTriggerHandler.